docs/book/src/SUMMARY.md

```diff
@@ -51,6 +51,7 @@
         - [Metadata propagation](./developer/architecture/controllers/metadata-propagation.md)
         - [Multi-tenancy](./developer/architecture/controllers/multi-tenancy.md)
         - [Support multiple instances](./developer/architecture/controllers/support-multiple-instances.md)
+        - [ClusterTopology](./developer/architecture/controllers/cluster-topology.md)
```

Comment:
nit: can you move it up a bit please. Maybe above or directly below MachinePool would be good (the last few docs are not really about individual controllers).